docs/gettinginvolved/plugins.rst

```diff
@@ -11,7 +11,7 @@ Block device backends
 =====================
 
 Flocker implements generic logic for network-based block device storage.
-Examples of such storage include Amazon EBS and EMC ScaleIO.
+Examples of such storage include :ref:`Amazon EBS<aws-dataset-backend>` and :ref:`EMC ScaleIO/XtremIO<emc-dataset-backend>`.
```

Comment:
This makes it seem like ScaleIO and XtremIO are just different names for the same thing. Maybe this needs an "and".